apps/coscrad-frontend/src/components/higher-order-components/filtered-categorizable-index-container.tsx

```diff
@@ -39,5 +44,15 @@ export const FilteredCategorizableIndexContainer = <
     // Wrap in error and pending presentation
     const Presenter = displayLoadableWithErrorsAndLoading(IndexPresenter);
 
-    return <Presenter {...filteredLoadableModels} />;
+    return (
+        <div>
+            <Presenter {...filteredLoadableModels} />
+            {loadableModels.data?.indexScopedActions && (
```

Comment:
Shouldn't we use the display loadable helper here?

Reply:
[Here](https://www.pivotaltracker.com/story/show/184107132) is a story for that.